README.md

```diff
@@ -1,19 +1,22 @@
 # DirectX Shader Compiler
 
-[![Build status](https://ci.appveyor.com/api/projects/status/6sx47j66g4dbyem9/branch/main?svg=true)](https://ci.appveyor.com/project/dnovillo/directxshadercompiler/branch/main)
-
 The DirectX Shader Compiler project includes a compiler and related tools used to compile High-Level Shader Language (HLSL) programs into DirectX Intermediate Language (DXIL) representation. Applications that make use of DirectX for graphics, games, and computation can use it to generate shader programs.
 
 For more information, see the [Wiki](https://github.com/microsoft/DirectXShaderCompiler/wiki).
 
 Visit the [DirectX Landing Page](https://devblogs.microsoft.com/directx/landing-page/) for more resources for DirectX developers.
 
 ## Downloads
-You can download the latest successful build's artifacts (built by Appveyor) for the main branch:
+
+> [!CAUTION]
+> Latest project builds are temporarily unavailable. See [Issue #6175](https://github.com/microsoft/DirectXShaderCompiler/issues/6175).
```

Comment:
Shall we redirect them to the Vulkan SDK?

Reply:
I don't think so, there's still the "Pre-built Releases" below for folks want a stable known-good rather than a latest tip-of-tree build. It's probably better to just remove this section entirely for now.